dokku.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+set -eu
+
+cd /tmp && wget https://github.com/casey/just/releases/download/1.40.0/just-1.40.0-x86_64-unknown-linux-musl.tar.gz && tar -xzf just-1.40.0-x86_64-unknown-linux-musl.tar.gz && chmod 555 just
+sudo mv /tmp/just /usr/bin/
```

Comment:
thought: While downloading this way is OK, this is because the default dev container image is based on Ubuntu 22.04, which doesn't have `just` in its repositories.